<!DOCTYPE html>
<html xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd" lang="en-gb" xml:lang="en-gb" data-mc-search-type="Stem" data-mc-help-system-file-name="index.xml" data-mc-path-to-help-system="../../../../../" data-mc-target-type="WebHelp2" data-mc-runtime-file-type="Topic" data-mc-preload-images="false" data-mc-in-preview-mode="false" data-mc-toc-path="Modules|Genetics|Configuration|Variants">
    <!-- saved from url=(0016)http://localhost -->
    <head>
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <meta charset="utf-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <link href="../../../../../Skins/Default/Stylesheets/Slideshow.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../../../../../Skins/Default/Stylesheets/TextEffects.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../../../../../Skins/Default/Stylesheets/Topic.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../../../../../Skins/Default/Stylesheets/Components/Styles.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../../../../../Skins/Default/Stylesheets/Components/Tablet.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../../../../../Skins/Default/Stylesheets/Components/Mobile.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <meta http_equiv="Content-Language" content="en-gb" /><title>Variants</title>
        <link href="../../../../Resources/StyleSheets/shared.css" rel="stylesheet" type="text/css" />
        <script src="../../../../../Resources/Scripts/jquery.min.js" type="text/javascript">
        </script>
        <script src="../../../../../Resources/Scripts/require.min.js" defer="defer" type="text/javascript">
        </script>
        <script src="../../../../../Resources/Scripts/require.config.js" defer="defer" type="text/javascript">
        </script>
        <script src="../../../../../Resources/Scripts/foundation.min.js" defer="defer" type="text/javascript">
        </script>
        <script src="../../../../../Resources/Scripts/plugins.min.js" defer="defer" type="text/javascript">
        </script>
        <script src="../../../../../Resources/Scripts/MadCapAll.js" defer="defer" type="text/javascript">
        </script>
    </head>
    <body>
        <h1><a name="aanchor2560"></a>
            <MadCap:concept term="Genetics" />Variants</h1>
        <p>This chapter presents </p>
        <ul>
            <li value="1">
                <p>the <a href="#Copy">Copy number variant</a> and <a href="#Sequence">Sequence variant</a> editors.</p>
                <p>These editors can be opened via <span class="ui_element">Start</span> -&gt; <span class="ui_element">Genetics</span> -&gt; <span class="ui_element">Configuration</span> -&gt; <span class="ui_element">Variants</span> -&gt; <span class="ui_element">Copy number variants</span>/<span class="ui_element">Sequence variants</span>.</p>
            </li>
            <li value="2">how to <a href="#Variant">approve/undo the approval</a> of a variant</li>
            <li value="3">how to get a <a href="#Families">list of families</a> in which a given variant has been found.</li>
            <li value="4">how to <a href="#Importin">import</a> lists of variants into the GLIMS database.</li>
        </ul>
        <div class="text_box_notes">
            <p><span style="color: #616E14; font-weight: bold;">Note</span>
            </p>
            <p>The functions that can be used to retrieve specific information on a chosen variant are described <a href="../mispl/variants(results).htm">here</a>.</p>
        </div>
        <h2><a name="Copy"></a>Copy number variants</h2>
        <p><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="variants_copy_number.png"><img class="MCPopupThumbnail img imgLargeImage" data-mc-width="1289" data-mc-height="916" src="variants_copy_number_thumb_0_120.png" tabindex="" /></a>
        </p>
        <p class="fieldlabel"><a name="CNVGenomicBuild"></a>Genomic build</p>
        <div class="indented">Indicates which genomic standard is used. For CNVs, <span class="ui_element">None</span> cannot be chosen.</div>
        <p class="fieldlabel"><a name="CNVLocus"></a>Locus</p>
        <div class="indented">Chromosome or chromosome region.</div>
        <div class="indented">Click the [...] button to open a list of chromosomes and chromosome regions, or use the <a href="../../routine/genetic_exam/g_e_screen_search_locus.htm">Add locus</a> button.</div>
        <p class="fieldlabel"><a name="Start"></a>Start position</p>
        <div class="indented">25 digits maximum</div>
        <p class="fieldlabel">Stop position</p>
        <div class="indented">25 digits maximum</div>
        <p class="fieldlabel">Size - kbp - Mbp</p>
        <div class="indented">Length of the variant: distance between the start and stop positions. Automatically calculated.</div>
        <p class="fieldlabel"><a name="CNVDNAChangeType"></a>Type</p>
        <div class="indented">Choose a DNA change type in the list. This list contains the active <a href="change_types.htm">DNA change type</a> records.
        <p>Used in particular for reporting and statistics.</p></div>
        <p class="fieldlabel">Copies</p>
        <div class="indented">Number of copies</div>
        <p class="fieldlabel">Ratio</p>
        <div class="indented">Free text</div>
        <p class="fieldlabel"><a name="GeneCNV"></a>Gene or gene region</p>
        <div class="indented">Loci of <a href="../genome/loci.htm#LocusType">type</a> <span class="ui_element">Genes</span> and <span class="ui_element">Gene regions</span> on which the CNV is located. Must refer to the same chromosome as the CNV.</div>
        <p class="fieldlabel">Default transcript</p>
        <div class="indented"><a href="../genome/genes.htm#Gene2">Default transcript</a> of the gene.</div>
        <p class="fieldlabel">Complementary information</p>
        <div class="indented"><a href="#Compleme">See below</a>.</div>
        <p class="fieldlabel">Codes</p>
        <div class="indented"><a href="#Codes">See below</a>.</div>
        <h2><a name="Sequence"></a>Sequence variants</h2>
        <p><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="variants_sequence.png"><img class="MCPopupThumbnail img imgLargeImage" data-mc-width="1288" data-mc-height="917" src="variants_sequence_thumb_0_120.png" tabindex="" /></a>
        </p>
        <p class="fieldlabel"><a name="SeqVarGenomicBuild"></a>Genomic build</p>
        <div class="indented">Indicates which genomic standard is used. For sequence variants, <span class="ui_element">None</span> may be selected.</div>
        <p class="fieldlabel"><a name="SeqVarLocus"></a>Locus</p>
        <div class="indented">Gene or gene region.</div>
        <div class="indented">Click the [...] button to open a list of genes and gene regions, or use the <a href="../../routine/genetic_exam/g_e_screen_search_locus.htm">Add loci</a> button.</div>
        <p class="fieldlabel"><a name="Transcript"></a>Transcript</p>
        <div class="indented">When a locus (gene or gene region) is indicated, this field is automatically filled in with the <a href="../genome/genes.htm#Gene2">default transcript</a> of the locus.</div>
        <div class="indented">        You can also<ul><li value="1">click the [...] button to open the list of all the transcripts of the selected locus.</li><li value="2">type the name of a transcript.</li></ul></div>
        <p class="fieldlabel"><a name="LRG"></a>LRG</p>
        <div class="indented"><a href="https://www.lrg-sequence.org/search/?query=*">Locus reference genomic id</a> (40 characters maximum).</div>
        <p class="fieldlabel">Start position</p>
        <div class="indented">Start position of the sequence variant, is also valid without a subsequent <span class="ui_element">Stop position</span>.</div>
        <p class="fieldlabel">Stop position</p>
        <div class="indented">Stop position of the sequence variant, not allowed without a preceding <span class="ui_element">Start position</span> and cannot be smaller than the start position.</div>
        <p class="fieldlabel">gDNA</p>
        <div class="indented">Description of the genomic DNA change (40 characters maximum).
        <div class="MCDropDown MCDropDown_Closed dropDown"><span class="MCDropDownHead dropDownHead"><a style="font-weight: bold; color: #003366" href="javascript:void(0)" class="MCDropDownHotSpot dropDownHotspot MCDropDownHotSpot_ MCHotSpotImage" aria-expanded="false"><img class="MCDropDown_Image_Icon" src="../../../../../Skins/Default/Stylesheets/Images/transparent.gif" height="13" width="16" alt="Closed" data-mc-alt2="Open" />Example</a></span><div class="MCDropDownBody dropDownBody"><div class="text_box_examples"><p><span style="color: #003366; font-weight: bold;">Example</span></p><p>It should contain the following elements in the following order:</p><ul><li value="1">"g."</li><li value="2">position numbering in relation to the first nucleotide of the genomic reference sequence</li><li value="3"><a href="change_types.htm#Usable">change</a>, e.g. "G&gt;A", "delA", "dupC"</li></ul><p>Example: g.12G&gt;A</p></div></div></div></div>
        <p class="fieldlabel"><a name="cDNA"></a>cDNA</p>
        <div class="indented">Description of the coding DNA change (40 characters maximum).
        <div class="MCDropDown MCDropDown_Closed dropDown"><span class="MCDropDownHead dropDownHead"><a style="font-weight: bold; color: #003366" href="javascript:void(0)" class="MCDropDownHotSpot dropDownHotspot MCDropDownHotSpot_ MCHotSpotImage" aria-expanded="false"><img class="MCDropDown_Image_Icon" src="../../../../../Skins/Default/Stylesheets/Images/transparent.gif" height="13" width="16" alt="Closed" data-mc-alt2="Open" />Example</a></span><div class="MCDropDownBody dropDownBody"><div class="text_box_examples"><p><span style="color: #003366; font-weight: bold;">Example</span></p><p>It should contain the following elements in the following order:</p><ul><li value="1">"c."</li><li value="2">position numbering in relation to the first nucleotide of the translation start codon of the coding DNA reference sequence</li><li value="3"><a href="change_types.htm#Usable">change</a>, e.g. "G&gt;A", "delA", "dupC"</li></ul><p>Example: c.76delA</p></div></div></div></div>
        <p class="fieldlabel"><a name="SeqVarDNAChangeType"></a>Type</p>
        <div class="indented">
            <p>Choose a DNA change type in the list. This list contains the active <a href="change_types.htm">DNA change type</a> records.</p>
            <p>Used in particular for reporting and statistics.</p>
        </div>
        <p class="fieldlabel"><a name="Amino"></a>Amino acid</p>
        <div class="indented">Description of the protein change (40 characters maximum).
        <div class="MCDropDown MCDropDown_Closed dropDown"><span class="MCDropDownHead dropDownHead"><a style="font-weight: bold; color: #003366" href="javascript:void(0)" class="MCDropDownHotSpot dropDownHotspot MCDropDownHotSpot_ MCHotSpotImage" aria-expanded="false"><img class="MCDropDown_Image_Icon" src="../../../../../Skins/Default/Stylesheets/Images/transparent.gif" height="13" width="16" alt="Closed" data-mc-alt2="Open" />Example</a></span><div class="MCDropDownBody dropDownBody"><div class="text_box_examples"><p><span style="color: #003366; font-weight: bold;">Example</span></p><p>It should contain the following elements in the following order:</p><ul><li value="1">"p."</li><li value="2">position numbering in relation to the first amino acid of the protein sequence</li><li value="3"><a href="change_types.htm#Description">change</a>, e.g. "G&gt;A", "delA", "dupC"</li></ul><p>Example: p.Met1ext-5</p></div></div></div></div>
        <p class="fieldlabel">Type</p>
        <div class="indented">Choose a protein change type in the list. This list contains the active <a href="change_types.htm#Protein">Protein change type</a> records.</div>
        <p class="fieldlabel">nDNA</p>
        <div class="indented">Description of the non-coding DNA change (40 characters maximum).
        <div class="MCDropDown MCDropDown_Closed dropDown"><span class="MCDropDownHead dropDownHead"><a style="font-weight: bold; color: #003366" href="javascript:void(0)" class="MCDropDownHotSpot dropDownHotspot MCDropDownHotSpot_ MCHotSpotImage" aria-expanded="false"><img class="MCDropDown_Image_Icon" src="../../../../../Skins/Default/Stylesheets/Images/transparent.gif" height="13" width="16" alt="Closed" data-mc-alt2="Open" />Example</a></span><div class="MCDropDownBody dropDownBody"><div class="text_box_examples"><p><span style="color: #003366; font-weight: bold;">Example</span></p><p>It should contain the following elements in the following order:</p><ul><li value="1">"n."</li><li value="2">position numbering in relation to the first nucleotide of the non-coding DNA reference sequence</li><li value="3"><a href="change_types.htm#Usable">change</a>, e.g. "G&gt;A", "delA", "dupC"</li></ul></div></div></div></div>
        <p class="fieldlabel">mDNA</p>
        <div class="indented">Description of the mitochondrial DNA change  (40 characters maximum).
        <div class="MCDropDown MCDropDown_Closed dropDown"><span class="MCDropDownHead dropDownHead"><a style="font-weight: bold; color: #003366" href="javascript:void(0)" class="MCDropDownHotSpot dropDownHotspot MCDropDownHotSpot_ MCHotSpotImage" aria-expanded="false"><img class="MCDropDown_Image_Icon" src="../../../../../Skins/Default/Stylesheets/Images/transparent.gif" height="13" width="16" alt="Closed" data-mc-alt2="Open" />Example</a></span><div class="MCDropDownBody dropDownBody"><div class="text_box_examples"><p><span style="color: #003366; font-weight: bold;">Example</span></p><p>It should contain the following elements in the following order:</p><ul><li value="1">"m."</li><li value="2">position numbering in relation to the first nucleotide of the mitochondrial DNA reference sequence</li><li value="3"><a href="change_types.htm#Usable">change</a>, e.g. "G&gt;A", "delA", "dupC"</li></ul></div></div></div></div>
        <p class="fieldlabel">RNA</p>
        <div class="indented">Description of the RNA change  (40 characters maximum).
        <div class="MCDropDown MCDropDown_Closed dropDown"><span class="MCDropDownHead dropDownHead"><a style="font-weight: bold; color: #003366" href="javascript:void(0)" class="MCDropDownHotSpot dropDownHotspot MCDropDownHotSpot_ MCHotSpotImage" aria-expanded="false"><img class="MCDropDown_Image_Icon" src="../../../../../Skins/Default/Stylesheets/Images/transparent.gif" height="13" width="16" alt="Closed" data-mc-alt2="Open" />Example</a></span><div class="MCDropDownBody dropDownBody"><div class="text_box_examples"><p><span style="color: #003366; font-weight: bold;">Example</span></p><p>It should contain the following elements in the following order:</p><ul><li value="1">"r."</li><li value="2">position numbering in relation to the first nucleotide of the <ul><li value="1"> translation start codon of the RNA reference sequence</li><li value="2">non-coding RNA reference sequence</li></ul></li><li value="3"><a href="change_types.htm#Usable">change</a>, e.g. "G&gt;A", "delA", "dupC"</li></ul><p>Example: r.45C&gt;U</p></div></div></div></div>
        <p class="fieldlabel">Complementary information</p>
        <div class="indented"><a href="#Compleme">See below</a>.</div>
        <p class="fieldlabel">Codes</p>
        <div class="indented"><a href="#Codes">See below</a>.</div>
        <h2><a name="Compleme"></a>Complementary information</h2>
        <p>The following fields appear in the editors of both copy number variants and sequence variants:</p>
        <p class="fieldlabel"><a name="Method"></a>Method</p>
        <div class="indented">Choose a variant method in the list. This list can be updated using the field <a href="../general_settings.htm#Variant">Variant method list</a>.</div>
        <p class="fieldlabel">Method info</p>
        <div class="indented">Free text field to add information about the method.</div>
        <p class="fieldlabel"><a name="Clinical"></a>Clinical classification</p>
        <div class="indented">Choose a variant classification in the list. This list can be updated as follows:
        <ol><li value="1">open the <a href="../../../../configuration/site_attributes/site_attributes_site_domains.htm">site domain</a> VariantClassification</li><li value="2"><p>modify the list of values</p><p><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="variants_variant_classification.png"><img class="MCPopupThumbnail img imgLargeImage" data-mc-width="775" data-mc-height="465" src="variants_variant_classification_thumb_0_120.png" tabindex="" /></a></p></li></ol><p>The clinical classification can also be indicated in routine.&#160;If the classification indicated in routine differs from the classification configured here, the configured classification can be updated under conditions described <a href="../../routine/results/result_screen.htm#CNV">here</a>.</p></div>
        <p class="fieldlabel">Approved by</p>
        <div class="indented">Name of the user who approved the variant</div>
        <p class="fieldlabel">Approved on</p>
        <div class="indented">Date and time of the approval. If this field is empty, the variant has not been approved yet.</div>
        <p class="fieldlabel">Comment</p>
        <div class="indented"><a href="../../../../configuration/properties/properties.htm#Purpose">Rich text</a> field for internal comments (read-only when the variant is approved).</div>
        <p class="fieldlabel">Found internally</p>
        <div class="indented">Select this option manually if the variant was found in a specimen managed in your lab with a LIS different from&#160;GLIMS. </div>
        <div class="indented">If a  variant has been detected in your lab using GLIMS, and is therefore stored as a confirmed or validated variant result of a genetic exam, the option <span class="ui_element">Found internally</span> is automatically activated and cannot be deactivated manually.</div>
        <p class="fieldlabel">Found outside community</p>
        <div class="indented">Select this option if the variant is known in other databases or networks in the world.</div>
        <p class="fieldlabel">Found in community</p>
        <div class="indented">Select this option if the variant is known on the network your laboratory is part of (such as the GeneInsight network VariantWire), independently of whether it was found in your lab or another lab of the network.</div>
        <p class="fieldlabel">Active</p>
        <div class="indented">Indicates whether the variant can be used in routine (selected in the <a href="../../routine/genetic_exam/g_e_screen_search_variant.htm">variant search screen</a>, for instance).</div>
        <h2><a name="Codes"></a>Codes</h2>
        <p>The <span class="ui_element">Codes</span> list is displayed in the editors of variants that have already been saved. It displays the codes used to refer to the variant in different <a href="../../../../configuration/coding_systems/coding_systems.htm">coding systems</a>. These coding systems can for instance represent the networks in which the variant has been found. </p>
        <p>Double-click on a code to open a web page containing information on the variant.</p>
        <p>The chapter <a href="variants_codes.htm">Variant codes</a> describes how to add codes to this list and link them to web pages.</p>
        <h2><a name="Annotati"></a>Annotations</h2>Read-only textual information on the variant. The content of this information field can be set or retrieved using <a href="../mispl/variants(results).htm#Variant.">MISPL functions</a>.
        <div class="text_box_examples"><p><span style="color: #003366; font-weight: bold;">Example</span></p><p> A list of comma-separated "key-value" pairs (nesting, i.e. lists within lists, is not supported):</p><p>{</p><p>"af": 0.1078, "afr_af": 0.087, "amr_af": 0.1297, "eas_af": 0.0813, "eur_af": 0.1481, "sas_af": 0.1063, "aa_af": 0.09623, "ea_af": 0.1543</p><p style="font-weight: bold;">}</p></div><h2><a name="Variant"></a>Variant approval</h2><p>Contextual menu/ribbon items, <span class="ui_element">Approve</span> and <span class="ui_element">Undo approval</span>, are displayed if </p><ul><li value="1">the user's role  is <span class="ui_element">System manager</span> or <span class="ui_element">Developer</span>, or</li><li value="2">the user has the <a href="../../../../security/security_privileges.htm">privilege</a> VariantApproval.</li></ul><p>Once a variant is approved, the fields of the editor can no longer be edited. To edit these fields, you first need to withdraw the approval.</p><h2><a name="Families"></a>Families linked to a variant</h2><p>Clicking <span class="ui_element">Families</span> in the ribbon of the <span class="ui_element">CNV</span> or <span class="ui_element">Sequence variant</span> editor opens a list of families in which the selected variant was found on at least one member. That is, at least one family member is linked to a confirmed or validated <a href="../../routine/results/variant_results.htm">variant result</a> for the selected variant.</p><h2><a name="Importin"></a>Importing variants</h2><p>Lists of variants can be imported using the <a href="../../../../application_management/import_export/general/import_export_denormalized.htm">denormalised export-import</a> function of GLIMS.</p></body>
</html>