<!DOCTYPE html>
<html xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd" lang="en-gb" xml:lang="en-gb" data-mc-search-type="Stem" data-mc-help-system-file-name="index.xml" data-mc-path-to-help-system="../../../../../" data-mc-target-type="WebHelp2" data-mc-runtime-file-type="Topic" data-mc-preload-images="false" data-mc-in-preview-mode="false" data-mc-toc-path="Modules|Genetics|Configuration|Genome">
    <!-- saved from url=(0016)http://localhost -->
    <head>
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <meta charset="utf-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <link href="../../../../../Skins/Default/Stylesheets/Slideshow.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../../../../../Skins/Default/Stylesheets/TextEffects.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../../../../../Skins/Default/Stylesheets/Topic.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../../../../../Skins/Default/Stylesheets/Components/Styles.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../../../../../Skins/Default/Stylesheets/Components/Tablet.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../../../../../Skins/Default/Stylesheets/Components/Mobile.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <meta http_equiv="Content-Language" content="en-gb" /><title>Gene codes</title>
        <link href="../../../../Resources/StyleSheets/shared.css" rel="stylesheet" type="text/css" />
        <script src="../../../../../Resources/Scripts/jquery.min.js" type="text/javascript">
        </script>
        <script src="../../../../../Resources/Scripts/require.min.js" defer="defer" type="text/javascript">
        </script>
        <script src="../../../../../Resources/Scripts/require.config.js" defer="defer" type="text/javascript">
        </script>
        <script src="../../../../../Resources/Scripts/foundation.min.js" defer="defer" type="text/javascript">
        </script>
        <script src="../../../../../Resources/Scripts/plugins.min.js" defer="defer" type="text/javascript">
        </script>
        <script src="../../../../../Resources/Scripts/MadCapAll.js" defer="defer" type="text/javascript">
        </script>
    </head>
    <body>
        <h1><a name="aanchor3411"></a>
            <MadCap:concept term="Genetics" />Gene codes</h1>
        <h2>Introduction</h2>
        <p>When an <a href="hgnc_import.htm">HGNC file</a> is imported, the HGNC code of each gene is saved automatically in the gene's <a href="genes.htm#Imported">HGNC id</a> field. However the HGNC file contains other gene codes, such as those of the <a href="http://www.omim.org/">OMIM compendium</a>. If you want (some of) these codes to be imported in GLIMS, you need to configure <a href="../../../../configuration/coding_systems/coding_systems.htm">coding systems</a> and code sets on the Gene table.</p>
        <h2>Configuration</h2>
        <p>The configuration consists in</p>
        <ol>
            <li value="1">creating a new coding system (for OMIM or pubmed, for instance)</li>
            <li value="2">
                <p>adding a code set for the Gene table to this coding system. The following fields of the Code set editor should be filled in:</p>
                <ul>
                    <li value="1">
                        <p><span class="ui_element">Table</span>: Gene</p>
                    </li>
                    <li value="2">
                        <p><span class="ui_element">Unique record</span>: unchecked</p>
                    </li>
                    <li value="3"><span class="ui_element">Unique value</span>: unchecked, otherwise the import will not work.</li>
                    <li value="4">
                        <p><span class="ui_element">URL</span>: URL of the website where information on the gene whose code belongs to the code set  can be found. The string must end with "{=.Value}", for instance https://www.omim.org/entry/{=.Value}.</p>
                        <p>Once the codes have been imported, GLIMS replaces this {=.Value} automatically with the code stored for the gene  when you click on <span class="ui_element">Open web page</span> in the <span class="ui_element">Codes</span> browser.</p>
                        <div class="text_box_examples">
                            <p><span style="color: #003366; font-weight: bold;">Example</span>
                            </p>
                            <p><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="codes_website.png"><img class="MCPopupThumbnail img imgLargeImage" data-mc-width="1131" data-mc-height="676" src="codes_website_thumb_0_120.png" tabindex="" /></a>
                            </p>
                        </div>
                    </li>
                    <li value="5">
                        <p><span class="ui_element">Import key</span>: import key provided in the import file, e.g "omim_id". This import key should only be used for the present code set.</p>
                        <div class="text_box_notes">
                            <p>This is a (possibly incomplete) list of import keys for the corresponding coding systems (more information on the <a href="https://www.genenames.org/help/statistics-and-downloads/">HGNC website</a>):</p>
                            <p><strong>entrez_id</strong>
                            </p>
                            <p>NCBI gene ID. Found within the "Gene resources" section of the gene symbol report.</p>
                            <p><strong>ensembl_gene_id</strong>
                            </p>
                            <p>Ensembl gene ID. Found within the "Gene resources" section of the gene symbol report.</p>
                            <p><strong>vega_id</strong>
                            </p>
                            <p>Vega gene ID. Found within the "Gene resources" section of the gene symbol report.</p>
                            <p><strong>ucsc_id</strong>
                            </p>
                            <p>UCSC gene ID. Found within the "Gene resources" section of the gene symbol report.</p>
                            <p><strong>ena</strong>
                            </p>
                            <p>International Nucleotide Sequence Database Collaboration (GenBank, ENA and DDBJ) accession number(s). Found within the "Nucleotide resources" section of the gene symbol report.</p>
                            <p><strong>ccds_id</strong>
                            </p>
                            <p>Consensus CDS ID. Found within the "Nucleotide resources" section of the gene symbol report.</p>
                            <p><strong>uniprot_ids</strong>
                            </p>
                            <p>UniProt protein accession. Found within the "Protein resource" section of the gene symbol report.</p>
                            <p><strong>pubmed_id</strong>
                            </p>
                            <p>Pubmed and Europe Pubmed Central PMID(s).</p>
                            <p><strong>mgd_id</strong>
                            </p>
                            <p>Mouse genome informatics database ID. Found within the "Homologs" section of the gene symbol report.</p>
                            <p><strong>rgd_id</strong>
                            </p>
                            <p>Rat genome database gene ID. Found within the "Homologs" section of the gene symbol report.</p>
                            <p><strong>lsdb</strong>
                            </p>
                            <p>The name of the Locus Specific Mutation Database and URL for the gene separated by a | character, e.g. Mutations of the ATP-binding Cassette Transporter Retina|http://www.retina-international.org/files/sci-news/abcrmut.htm</p>
                            <p><strong>cosmic</strong>
                            </p>
                            <p>Symbol used within the Catalogue of somatic mutations in cancer for the gene.</p>
                            <p><strong>omim_id</strong>
                            </p>
                            <p>Online Mendelian Inheritance in Man (OMIM) ID</p>
                            <p><strong>mirbase</strong>
                            </p>
                            <p>miRBase ID</p>
                            <p><strong>homeodb</strong>
                            </p>
                            <p>Homeobox Database ID</p>
                            <p><strong>snornabase</strong>
                            </p>
                            <p>snoRNABase ID</p>
                            <p><strong>bioparadigms_slc</strong>
                            </p>
                            <p>Symbol used to link to the SLC tables database at bioparadigms.org for the gene</p>
                            <p><strong>orphanet</strong>
                            </p>
                            <p>Orphanet ID</p>
                            <p><strong>pseudogene.org</strong>
                            </p>
                            <p>Pseudogene.org ID</p>
                            <p><strong>horde_id</strong>
                            </p>
                            <p>Symbol used within HORDE for the gene</p>
                            <p><strong>merops</strong>
                            </p>
                            <p>ID used to link to the MEROPS peptidase database</p>
                            <p><strong>imgt</strong>
                            </p>
                            <p>Symbol used within international ImMunoGeneTics information system</p>
                            <p><strong>iuphar</strong>
                            </p>
                            <p>The objectId used to link to the IUPHAR/BPS Guide to PHARMACOLOGY database. To link to IUPHAR/BPS Guide to PHARMACOLOGY database only use the number (only use 1 from the result objectId:1) in the example URL.</p>
                            <p><strong>kznf_gene_catalog</strong>
                            </p>
                            <p>ID used to link to the Human KZNF Gene Catalog</p>
                            <p><strong>mamit-trnadb</strong>
                            </p>
                            <p>ID to link to the Mamit-tRNA database</p>
                            <p><strong>cd</strong>
                            </p>
                            <p>Symbol used within the Human Cell Differentiation Molecule database for the gene</p>
                            <p><strong>lncrnadb</strong>
                            </p>
                            <p>lncRNA Database ID</p>
                            <p><strong>enzyme_id</strong>
                            </p>
                            <p>ENZYME EC accession number</p>
                            <p><strong>intermediate_filament_db</strong>
                            </p>
                            <p>ID used to link to the Human Intermediate Filament Database</p>
                        </div>
                    </li>
                </ul>
            </li>
            <li value="3">
                <p>Now the codes will be saved in GLIMS when an HGNC file is imported.</p>
                <p><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="codes_editor.png"><img class="MCPopupThumbnail img imgLargeImage" data-mc-width="485" data-mc-height="471" src="codes_editor_thumb_0_120.png" tabindex="" /></a>
                </p>
                <p class="fieldlabel">Table</p>
                <div class="indented">Gene</div>
                <p class="fieldlabel">Record</p>
                <div class="indented">Approved symbol of the gene</div>
                <p class="fieldlabel">Value</p>
                <div class="indented">Code given to the gene symbol in the given coding system.</div>
                <p>The other fields are described <a href="../../../../configuration/coding_systems/coding_systems.htm#Code">here</a>.</p>
                <div class="text_box_notes">
                    <p><span style="color: #616E14; font-weight: bold;">Note</span>
                    </p>
                    <p>When codes existing in GLIMS are not present in the newest import file, they are deleted from the GLIMS database automatically.</p>
                </div>
                <p>&#160;</p>
            </li>
        </ol>
        <h2>Routine</h2>
        <p>Once coding systems and code sets are created, the list of codes linked to a given gene can be </p>
        <ul>
            <li value="1">opened by clicking <a href="../../../../configuration/coding_systems/coding_systems.htm#Direct_access_to_all_codes">Codes</a> in the ribbon tab <span class="ui_element">Tools</span> of the <span class="ui_element">Gene</span> editor or browser,</li>
            <li value="2">displayed in the <a href="../../../../user_interface/user_interface_browsers.htm#Coding_systems">browser</a>:</li>
        </ul>
        <p><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="codes_list.png"><img class="MCPopupThumbnail img imgLargeImage" data-mc-width="920" data-mc-height="638" src="codes_list_thumb_0_120.png" tabindex="" /></a>
        </p>
    </body>
</html>