<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd" lang="en-gb" xml:lang="en-gb" data-mc-search-type="Stem" data-mc-help-system-file-name="index.xml" data-mc-path-to-help-system="../../../" data-mc-target-type="WebHelp2" data-mc-runtime-file-type="Topic" data-mc-preload-images="false" data-mc-in-preview-mode="false" data-mc-toc-path="">
    <!-- saved from url=(0016)http://localhost -->
    <head>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" xmlns="" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <meta charset="utf-8" />
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <link href="../../../Skins/Default/Stylesheets/Slideshow.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../../../Skins/Default/Stylesheets/TextEffects.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../../../Skins/Default/Stylesheets/Topic.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../../../Skins/Default/Stylesheets/Components/Styles.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../../../Skins/Default/Stylesheets/Components/Tablet.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
        <link href="../../../Skins/Default/Stylesheets/Components/Mobile.css" rel="stylesheet" type="text/css" data-mc-generated="True" /><title>Release notes 9.9.7 - Genetics REST: aminoAcidChange field no longer required in Variant JSON + support for automatic gene region creation for sequence variants (GLIMS_GENX_LAB-00936)</title>
        <script type="text/javascript" src="../../Resources/hyperlink_target.js">
        </script>
        <link href="../../Resources/StyleSheets/shared.css" rel="stylesheet" type="text/css" />
        <script src="../../../Resources/Scripts/jquery.min.js" type="text/javascript">
        </script>
        <script src="../../../Resources/Scripts/require.min.js" defer="defer" type="text/javascript">
        </script>
        <script src="../../../Resources/Scripts/require.config.js" defer="defer" type="text/javascript">
        </script>
        <script src="../../../Resources/Scripts/foundation.min.js" defer="defer" type="text/javascript">
        </script>
        <script src="../../../Resources/Scripts/plugins.min.js" defer="defer" type="text/javascript">
        </script>
        <script src="../../../Resources/Scripts/MadCapAll.js" defer="defer" type="text/javascript">
        </script>
    </head>
    <body>
        <h1><a name="aanchor1991"></a>
            <MadCap:concept term="Release notes" />
            <MadCap:conditionalText data-mc-conditions="Primary.10_webhelp_only">Release notes 9.9.7 - </MadCap:conditionalText>Genetics REST: aminoAcidChange field no longer required in Variant JSON + support for automatic gene region creation for sequence variants (GLIMS_GENX_LAB-00936)</h1>
        <h2>Issues</h2>
        <ol>
            <li value="1">The J_GENX_RI translator gives an error when the aminoAcidChange field is not specified in the input field as it is marked as a mandatory field. </li>
            <li value="2">When a sequence variant is created via the REST service, GLIMS passes the name of the locus. However, certain customers define their own gene regions on top of the ones that GLIMS imported via HGNC. As a result, it is possible that the locus does not exist and GLIMS returns an error.</li>
        </ol>
        <h2>Solution</h2>
        <p>These issues, which occurred since GLIMS 9.9.0, have been corrected:</p>
        <ol>
            <li xmlns="" value="1">The aminoAcidChange field is now removed from the list of mandatory fields, and not specifying the field will no longer result in a JSON import error.</li>
            <li xmlns="" value="2">
                <p xmlns="http://www.w3.org/1999/xhtml">GLIMS will now check to see if the locus name corresponds to a gene region. If this is the case, GLIMS will try to create that gene region on the fly.</p>
                <div class="text_box_notes" xmlns="http://www.w3.org/1999/xhtml">
                    <p><span style="color: #616E14; font-weight: bold;">Note</span>
                    </p>
                    <p>Loci of type 'Gene region' must adhere to the following naming format:</p>
                    <p>"Gene:GeneRegionTypeNumber"</p>
                    <p>E.g.: CFTR:Intron1, BRCA1:Exon3</p>
                    <p>GeneRegionType is one of the following entries (case sensitive!):</p>
                    <ul>
                        <li value="1"> Exon</li>
                        <li value="2">Intron</li>
                        <li value="3">5'UTR</li>
                        <li value="4">3'UTR</li>
                        <li value="5">Amplicon</li>
                    </ul>
                    <p>If the user specifies a wrong GeneRegionType, an error message is shown and the variant is not created/updated.</p>
                </div>
            </li>
        </ol>
    </body>
</html>